scripts/utils/vermoji.ts

```diff
@@ -316,12 +317,23 @@ const VERMOJIS = [
   '🧀',
 ];
 
+// filter out the 'unknown version vermoji'
+vermojis = vermojis.filter((vermoji) => vermoji !== UNKNOWN_VERMOJI);
+
 export function getVermoji(changelogPath: string) {
   const changelog = fs.readFileSync(changelogPath, 'utf8');
 
   while (true) {
-    const vermoji = VERMOJIS[Math.round(Math.random() * (VERMOJIS.length - 1))];
-    if (!changelog.includes(vermoji)) {
+    const randomIndex = Math.round(Math.random() * (vermojis.length - 1));
```

Comment:
I know the existing code was like that, but the canonical way to pick a random index is:
```suggestion
    const randomIndex = Math.floor(Math.random() * vermojis.length);
```
Using `Math.round()` is unfair bias against the first and last element.

Reply:
👍 https://github.com/ionic-team/stencil/pull/3265/commits/24aa2d4ae0e3201b23b2f2172624462f831944d7